From aafa3512b7b742f5e66a5543e41974cc5e7eebfa Mon Sep 17 00:00:00 2001
From: maxice8 <thinkabit.ukim@gmail.com>
Date: Sun, 22 Jul 2018 18:54:17 -0300
Subject: [PATCH] pcap-usb-linux.c: add missing limits.h for musl systems.

fix compilation on musl libc systems like Void Linux and Alpine.

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Retrieved from:
https://github.com/the-tcpdump-group/libpcap/commit/aafa3512b7b742f5e66a5543e41974cc5e7eebfa]
---
 pcap-usb-linux.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pcap-usb-linux.c b/pcap-usb-linux.c
index 6f8adf65e..b92c05ea1 100644
--- a/pcap-usb-linux.c
+++ b/pcap-usb-linux.c
@@ -50,6 +50,7 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <fcntl.h>
+#include <limits.h>
 #include <string.h>
 #include <dirent.h>
 #include <byteswap.h>
